lib/preview/application.ex

```diff
@@ -38,6 +42,14 @@ defmodule Preview.Application do
     end
   end
 
+  if Application.get_env(:preview, :package_updater_impl) == Preview.Package.LocalUpdater do
```

Comment:
Instead of this conditional in the application.ex file we can fill the ETS table here: https://github.com/hexpm/preview/blob/main/lib/preview/package/local_updater.ex#L9.

Reply:
Done thanks